src/main.cpp

```diff
@@ -723,7 +722,7 @@ int wmain(void){
 		converter = w2xconv_init_with_processor(proc, cmdNumberOfJobs.getValue(), verbose);
 	}
 	else {
-		converter = w2xconv_init(gpu, cmdNumberOfJobs.getValue(), verbose);
+		converter = w2xconv_init(gpu, cmdNumberOfJobs.getValue(), verbose, /*keep_kernel=*/ false);
```

Comment:
Why these inline comments if you do not use them everywhere, like in `w32-apps/runbench.c#L39`